server.js

```diff
@@ -16,28 +32,33 @@ server.listen(3000, ()=>{
 
 server.use(bodyParser.json());
 
+//Initializo arrays
+cuentas = listarCuentas();
+usuarios = [];
+
+//RUTAS
 server.get("/cuentas", (req,res,err)=>{
-    res.json(cuentas);
+  res.json(cuentas);
 });
 
 server.get("/cuentas/:id", (req, res, err) => {
     const indice = cuentas.findIndex((element) => {
-        return element.nroCuenta == req.params.id;
+        return element.cuenta_id == req.params.id;
```

Comment:
o vamos usar las variables en camel case holaJuan o con guion bajo hola_juan, pero no mezclemos, solo una de las dos.

Reply:
Lo noté, pero traté de mantener los nombres de las tablas para no marearme. Tuve que cambiar todo el código porque en el código original era nroCuenta y luego en las tablas le cambiaron los nombres de los campos 🤷🏻‍♂️
No sé bien cual es la guía de estilo o regla cuando tenes en la base de datos guiones y en el código venis usando camelCase.